OpenEdXMobile/src/main/java/org/edx/mobile/util/CourseDateUtil.kt

```diff
@@ -0,0 +1,47 @@
+package org.edx.mobile.util
+
+import android.content.Context
+import android.text.TextUtils
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
+import org.edx.mobile.R
+import org.edx.mobile.model.course.CourseBannerInfoModel
+import org.edx.mobile.model.course.CourseBannerType
+
+/**
+ * Utils class for CourseDates
```

Comment:
`Utils class to populate the CourseDates banner view` I think is more appropriate.